dbt/models/qc/schema.yml

```diff
@@ -495,12 +629,32 @@ models:
     tests:
       - expression_is_true:
           name: qc_vw_change_in_high_low_value_sales_ratio_high
-          expression: |
-            price_less_than_10k_growth_status != 'No significant change'
```

Comment:
Note that I think this test previously had the logic backwards; `expression` should be treated [as an assert statement](https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#expression_is_true-source), but here it seems to be the opposite. There are a few places in this file where I had to make similar switches; another argument against using `expression_is_true`, since clearly it's confusing to remember how the expression works.